app/models/space.rb

```diff
@@ -92,11 +94,21 @@ def cleaned_graph
     {'metrics' => Array.wrap(cleaned_metrics), 'guesstimates' => Array.wrap(cleaned_guesstimates)}
   end
 
+  def visible_to?(user)
+    is_public? || user.id == self.user_id || user.member_of?(self.organization_id)
+  end
+
   def copy(user)
     space = Space.new(self.attributes.slice('name', 'description', 'graph'))
     space.user = user
     space.copied_from_id = self.id
     space.is_private = user.prefers_private?
+
+    # This space should be retained within the organization if it is being copied within the organization.
```

Comment:
So for now, if it's not part of an organization, it can't become one?  That's fine, just want to understand this.